algorithms/linfa-ftrl/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+#![doc = include_str!("../README.md")]
+
+mod algorithm;
+mod error;
+mod hyperparams;
+
+use crate::hyperparams::FtrlValidParams;
+pub use algorithm::Result;
+pub use error::FtrlError;
+pub use hyperparams::FtrlParams;
+use linfa::Float;
+use ndarray::Array1;
+use ndarray_rand::RandomExt;
+use rand::distributions::Uniform;
+
+#[cfg_attr(
+    feature = "serde",
+    derive(Serialize, Deserialize),
+    serde(crate = "serde_crate")
+)]
+#[derive(Clone, Debug)]
+pub struct FTRL<F: Float> {
```

Comment:
Hi, thanks for your contribution! @YuhanLiin nitpicking: should not it be `Ftrl` instead of `FTRL` as we already have `Pls`, `Pca` or `Svm` in `linfa`?

Reply:
I can't really find any counterexamples so yeah, it makes sense to name this `Ftrl`.